etc/ROS2TEST_QOS_PROFILES.xml

```diff
@@ -5,7 +5,7 @@ xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/5.3.1/rti_dds_qos
   <!-- ## Ros2TestQosLibrary -->
   <!-- ################################################################### -->
   <qos_library name="Ros2TestQosLibrary">
-    <qos_profile name="Ros2TestDefaultQos" is_default_qos="true">
+    <qos_profile name="Ros2TestDefaultQos" base_name="BuiltinQosLib::Baseline.5.3.0" is_default_qos="true">
```

Comment:
This is not really needed, as I think it's the default in Connext 5.1.x.

In case that a version of `rmw_connext` with https://github.com/ros2/rmw_connext/pull/406 corrected isn't available, the base name can be replaced with `Generic.510TransportCompatibility`, and also load that profile in the rti tools being use following https://community.rti.com/kb/how-do-i-use-tools-analyzer-monitor-admin-console-changed-transport-settings.